tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/rules/aux/Oracle.scala

```diff
@@ -12,6 +15,14 @@ import at.forsyte.apalache.tla.lir.TlaEx
  */
 trait Oracle extends Serializable {
 
+  /**
+   * The number of values that this oracle is defined over: `0..(size - 1)`.
```

Comment:
I don't understand what `0..(size - 1)` is indicating here. Is it supposed to be an enumeration of all the values? Does that mean it's only defined over integers?

Reply:
Oracles always return an integer value (in a given range). The interpretation of the value depends on the kind of oracle, but typically, `n` means e.g. choose the `n`-th `has` edge in the arena.

Reply:
Yes, oracles are always enumerating over integers. As Jure wrote, these are basically the indices of the edges. How they are mapped to SMT, depends on the implementation.

Reply:
I see. Thank you for clafifying!